dotnet/src/Connectors/Connectors.AI.OpenAI/ChatCompletion/AzureOpenAIChatCompletionService.cs

```diff
@@ -19,6 +19,21 @@ namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.ChatCompletion;
 /// </summary>
 public sealed class AzureOpenAIChatCompletionService : IChatCompletionService, ITextGenerationService
 {
+    #region BreakingGlass Properties
+    /// <summary>
+    /// Model Id for OpenAI or Deployment Name when using Azure OpenAI.
+    /// </summary>
+    public string DeploymentOrModelName => this._core.DeploymentOrModelName;
+
+    /// <summary>
+    /// OpenAI / Azure OpenAI Client
```

Comment:
At this point it'd be good to be writing the XML comments that we expect developers to actually see, e.g.
```suggestion
    /// Gets the underlying <see cref="OpenAIClient"/> used by this <see cref="AzureOpenAIChatCompletionService"/> instance.
```

Reply:
Resolved, applied commit.